docs/changelog.rst

```diff
@@ -20,24 +20,24 @@ Features - 4.0.0a10
   for packages passed in via :ref:`--installpkg <tox-run---installpkg>` - by :user:`gaborbernat`. (`#2204 <https://github.com/tox-dev/tox/issues/2204>`_)
 - Add support for rewriting script invocations that have valid shebang lines when the ``TOX_LIMITED_SHEBANG`` environment
   variable is set and not empty - by :user:`gaborbernat`. (`#2208 <https://github.com/tox-dev/tox/issues/2208>`_)
-- Support for the ``--discover`` CLI flag - by :user:`gaborbernat`. (`#2245 <https://github.com/tox-dev/tox/issues/2245>`_)
+- Support for the ``--discover`` CLI flag - by :user:`gaborbernat`. (`#2245 <https://github.com/tox-dev/tox/pull/2245>`_)
 - Moved the python packaging logic into a dedicate package `pyproject-api <https://pypi.org/project/pyproject-api>`_ and
-  use it as a dependency - by :user:`gaborbernat`. (`#2274 <https://github.com/tox-dev/tox/issues/2274>`_)
-- Drop python 3.6 support - by :user:`gaborbernat`. (`#2275 <https://github.com/tox-dev/tox/issues/2275>`_)
+  use it as a dependency - by :user:`gaborbernat`. (`#2274 <https://github.com/tox-dev/tox/pull/2274>`_)
+- Drop python 3.6 support - by :user:`gaborbernat`. (`feature #2275 <https://github.com/tox-dev/tox/pull/2275>`_)
```

Comment:
Where is `feature`coming from? And what it is for?

Reply:
When you add two such hyperlinks (PRs adding both two or more of bugfix, doc, feature at the same time) sphinx complains that you defined two references with the same key, this differentiates it.